### ISSUE TITLE
Download links for trained multi-agent intersection policies (CARLA / MACAD-Gym)

### ISSUE THREAD

**Issue by tahabinkashif** [praveen-palanisamy/macad-gym#92]
Request: Download links for trained multi-agent intersection policies (CARLA / MACAD-Gym)

Hi MACAD team, thanks for open-sourcing MACAD-Gym and MACAD-Agents!

TL;DR
Could you please share: direct download links (or a Git LFS/release asset path) to the pretrained multi-agent policies you mention for Urban Intersection scenarios in CARLA? I’m specifically looking for Two Independent Agents operating in a shared intersection environment (e.g., stop-sign / 3-way urban intersection).

---

What I need
-Trained weights/checkpoints for Two Separate Agents (non-communicating is fine) that were trained in CARLA on intersection tasks via MACAD-Gym environments.
- Load/eval script or command** to run rollouts (no training).
- Any config files (env IDs, scenario JSON, algo hyperparams) used for those weights.

Preferred formats: 
- PyTorch (`.pt`/`.pth`) or TF (`.ckpt`), zipped is fine.
- A short README snippet on how to load each agent in evaluation mode.

---

Environment assumptions (please correct if needed)
- CARLA: `0.9.x`
- MACAD-Gym: environment IDs such as `HomoNcomIndePOIntrxMASS3CTWN3-v0` (or the closest intersection env you used)
- MACAD-Agents: PPO/IMPALA multi-agent runners
- OS: Ubuntu 20.04/22.04, Python 3.8–3.10
- GPU: CUDA 11.x (optional for eval)

---

Minimal “play/eval” I hope to run
1) Start CARLA server (headless or display).
2) `conda/venv` with macad-gym + macad-agents requirements.
3) Download checkpoints (two agents).
4) Run an **eval/rollout** command like:
   ```bash
   python eval_multiagent.py \
     --env-id HomoNcomIndePOIntrxMASS3CTWN3-v0 \
     --agent-a-checkpoint /path/to/agent_A.ckpt \
     --agent-b-checkpoint /path/to/agent_B.ckpt \
     --episodes 10 --render
